pombola/core/management/commands/core_sync_everypolitician_uuid.py

```diff
@@ -0,0 +1,54 @@
+from everypolitician import EveryPolitician
+
+from django.core.management.base import BaseCommand
+
+from pombola.core.models import Person
+
+
+class Command(BaseCommand):
+    help = 'Sync EveryPolitician UUID to Person.everypolitician_uuid field'
+
+    def add_arguments(self, parser):
+        parser.add_argument('everypolitician_countries_json_git_ref',
+                            help="A git ref from the everypolitician-data repo")
+
+    def handle(self, **options):
+        verbose_level = options['verbosity']
+
+        # FIXME: This SHA is currently pointing at a branch which has the
+        # `peoples_assembly` identifier, but this isn't on master because
+        # South Africa is broken in EveryPolitician at the moment.
+        #
+        # TODO: Make this SHA configurable.
+        url_template = ('https://cdn.rawgit.com/everypolitician/everypolitician-data/'
+                        '{git_ref}/countries.json')
+
+        url = url_template.format(git_ref=options['everypolitician_countries_json_git_ref'])
+
+        ep = EveryPolitician(countries_json_url=url)
+        south_africa_assembly = ep.country('South-Africa').legislature('Assembly').popolo()
+
+        id_lookup = {}
+        for popolo_person in south_africa_assembly.persons:
+            pa_ids = [i for i in popolo_person.identifiers if i['scheme'] == 'peoples_assembly']
+            if pa_ids == []:
+                continue
+            id_lookup[pa_ids[0]['identifier']] = popolo_person.id
+
+        error_msg = u"No EveryPolitician UUID found for {} {} https://www.pa.org.za/person/{}/\n"
```

Comment:
This is partly just "fun with Python's `.format`" but, you could make this:
```
u"No EveryPolitician UUID found for {0.id} {0.name} https://www.pa.org.za/person/{0.slug}/\n"
```
... and then do `error_msg.format(person)`. It's a tiny thing, but it'd make changing the format string later less prone to error.

Reply:
😮 Wow, didn't know `.format` could do that!! You're quite right, that's less error prone. Have switched to using that syntax.